templates/consonant/blocks/marquee/marquee.css

```
@@ -519,7 +519,7 @@ main .marquee.center .marquee-column:not(.marquee-image) {
     main .marquee.large .background img,
     main .marquee.large .background video {
         max-height: unset;
-        width: 100%;
+        width: 101%;
```

```suggestion
        width: 100%;
        transform: none;
```
I am ok either way but, you could just cancel the transform instead.